### PR TITLE
Ensured Rating Data dialog output correct

### PR DESCRIPTION
**Pull request by derekagorhom**
fixes partly #8104

@Patowhiz , @lloyddewit , @N-thony 
This PR is ready for review

### CODE REVIEW

File: instat/dlgRatingScales.vb [hunk 1]
```diff
@@ -76,7 +76,7 @@ Public Class dlgRatingScales
         ucrPnlGraphType.AddToLinkedControls(ucrChkFlip, {rdoLikert, rdoStacked}, bNewLinkedHideIfParameterMissing:=True)
         ucrPnlGraphType.AddToLinkedControls(ucrNudNeutralLevel, {rdoLikert}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True, objNewDefaultState:=1)
         ucrPnlGraphType.AddToLinkedControls(ucrChkNumberOfCategories, {rdoLikert}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
-        ucrPnlGraphType.AddToLinkedControls(ucrSaveGraph, {rdoStacked, rdoLikert}, bNewLinkedHideIfParameterMissing:=True)
+        'ucrPnlGraphType.AddToLinkedControls(ucrSaveGraph, {rdoStacked, rdoLikert}, bNewLinkedHideIfParameterMissing:=True)
```
Comment (lloyddewit) [inline instat/dlgRatingScales.vb:79]: better to delete rather than comment?
```suggestion
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Comment by derekagorhom** [comment]
@lloyddewit i have made the changes, thank you

**Review by lloyddewit — APPROVED** [review]
@derekagorhom Thanks for the tidy/readable code, looks good to me

**Review by berylwaswa — APPROVED** [review]
Looks good.

**Review by lloyddewit — APPROVED** [review]

**Pull request merged**